jherey/WeConnect
# Bg fix redux state mutation 158609836

#### What does this PR do?
Changes redux state mutation to meet best practices

#### Description of Task to be completed?
- Add types constant to actions
- Add initial state to all reducers
- Change redux structure
- Fix failing server-side tests

#### How should this be manually tested?
- Go to application URL
- Open the redux dev tools
- Verify that previous state is not mutated
 
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#158609836

#### Screenshots (if appropriate)
N/A
